readthedocs/subscriptions/managers.py

```diff
@@ -218,3 +218,29 @@ def get_feature(self, obj, type):
             plan__subscriptions__organization=organization,
         )
         return feature.first()
+
+    # pylint: disable=redefined-builtin
+    def get_feature_value(self, obj, type, default=None):
+        """
+        Get the value of the given feature.
+
+        Use this function instead of ``get_feature().value``
+        when you need to respect the ``RTD_ALL_FEATURES_ENABLED`` setting.
+        """
+        if not settings.RTD_ALL_FEATURES_ENABLED:
+            feature = self.get_feature(obj, type)
+            if feature:
+                return feature.value
+        return default
```

Comment:
Here is where I think it needs to encapsulate more logic, so we can rely 100% on it without mixing the `.get_feature_value` with other logic in the code.